lib/puppet/functions/libkv/exists.rb

```diff
@@ -1,86 +1,119 @@
-# vim: set expandtab ts=2 sw=2:
+# Returns whether the `key` exists in the configured backend.
+#
+# @author https://github.com/simp/pupmod-simp-libkv/graphs/contributors
 #
-# @author Dylan Cochran <dylan.cochran@onyxpoint.com>
 Puppet::Functions.create_function(:'libkv::exists') do
```

Comment:
Not sure what namespace would make sense here. `libkv::key::exists` or the like?

It might make sense for the top level primitives to always mean `value` since that's what the 90% usage will be.